projects/plugins/wpcomsh/wpcom-features/class-wpcom-features.php

```diff
@@ -1149,22 +1150,25 @@ class WPCOM_Features {
 		self::STATS_FREE                        => array(
 			self::JETPACK_STATS_PLANS,
 			self::JETPACK_GROWTH_PLANS,
-		),
-		self::STATS_PAID                        => array(
 			array(
 				'before' => '2024-01-09',
-				self::WPCOM_PERSONAL_PLANS,
 				self::WPCOM_ALL_SITES,
 			),
+		),
+		self::STATS_PAID                        => array(
+			self::WPCOM_PERSONAL_AND_HIGHER_PLANS,
 			self::WP_P2_PLUS_MONTHLY,
-			self::WPCOM_PREMIUM_AND_HIGHER_PLANS,
 			self::JETPACK_STATS_PWYW,
 			self::JETPACK_STATS_MONTHLY,
 			self::JETPACK_STATS_BI_YEARLY,
 			self::JETPACK_STATS_YEARLY,
 			self::JETPACK_COMPLETE_PLANS,
+			self::JETPACK_BUSINESS_PLANS,
```

Comment:
@kangzj this was missing from the Jetpack version of this file, does it actually need adding here or does in fact need removing from the dotcom side?

Reply:
I'm not entirely sure 🤔

Reply:
Thanks for the ping, @kangzj. Based on the description in D158049-code, it seems like it actually needs adding here. Thanks for fixing that.

Reply:
It looks like odyssey stats sidesteps this class on atomic sites so it being missing doesn't matter, I'll sync this PR up to whatever we do in D167647-code for completeness but it should be a noop.